src/AdsList.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Grid from "@mui/material/Grid";
+import AdsCards from './AdsCards';
+import aparts from './data';
+
+    const AdsList = (props) => {
+        
+const {filtrAp} = props;
```

Comment:
Destructure right when you're getting props, and get all the filters by one destructurization instead of 5